### ISSUE TITLE
projectiles don't calculate extra damage modifiers correctly.

### ISSUE THREAD

**Issue by EnderiumSmith** [SlimeKnights/TinkersConstruct#2655]
the modifiers sharp, smite and bane of arthropods seem to apply the damage after the projectiles hits.

if i take a steel crossbow with manylyn bolts and add quarts it will do about 6 dmg more than without quarts, which is insignificant. blaze would do more.

if i use smite instead the crossbow will do less damage per hit than a rapier. it should deal at least a third of the Withers health.

**Comment by bonii-xx** [comment]
Can you rephrase that to be more clear? Of course projectiles deal damage after they hit something...?

**Comment by LovelyFlare** [comment]
Is this 1.10.2 or 1.7.10?

**Comment by EnderiumSmith** [comment]
1.10.2
@bonii-xx the bolts say they do about 10 damage but the actual damage varies depending on the bow limb.
if i add quartz III it says they do 15 damage but the actual damage is always 5 more than without quartz.
if i add smite III it adds 24 damage no matter what crossbow i use, so a smite IV rapier does more damage per hit. 

thats the problem: the damage added by these 3 modifiers should vary depending on the bow limb like the normal damage of the bolts. a steel crossbow should take at least 1/3 of the withers health with smite  III to be worth using.

**Comment by TherminatorX** [comment]
@EnderiumSmith The Wither has armor. The rapier bypasses armor.

**Comment by EnderiumSmith** [comment]
@TherminatorX so does the crossbow. the rapier can get smite IV with a paper piece. bolts can only have 3 modifiers.

**Comment by bonii-xx** [comment]
>thats the problem: the damage added by these 3 modifiers should vary depending on the bow limb like the normal damage of the bolts. a steel crossbow should take at least 1/3 of the withers health with smite III to be worth using.

No they should not. The damage added by the crossbow limb works the same as the damage added by quartz and smite. Additive. Maybe the base damage of bolts is displayed incorrectly on them.

**Comment by EnderiumSmith** [comment]
@bonii-xx wait, there's no multiplier? i thought it works like a tool rod with both a multiplier and a flat damage. in tic1 it was a multiplier. why not in tic2? this way a rapier does more damage PER HIT than a crossbow. kinda makes them worthless.

**Comment by bonii-xx** [comment]
Sounds more like the crossbow you crafted simply sucks.

**Comment by EnderiumSmith** [comment]
its a steel crossbow with manylyn bolts. the extra level of smite on the rapier adds more damage than the steel bow limb. bolts cant have paper parts or silver tool rods or potions of strength.

why cant there be a multiplier too? like the tool rod.

**Comment by bonii-xx** [comment]
A steel crossbow with pretty much anything oneshots zombies. Adding smite to bolts increases the damage exactly as much as adding smite to the rapier per hit. I still have no idea what you are talking about.

**Comment by EnderiumSmith** [comment]
the rapier can have paper parts for an extra level of smite.

 the crossbow is worthless since it takes 2 extra inventory slots for not very much damage. a longbow can also 1-shot normal mobs. 

a smite 3 bold should be able to do at least 100 smite damage with the most damaging bow limb.

ranged weapons should have both a multiplier and flat damage like a tool rod does for durability. its a crossbow, it supposed to have very high damage but low DPS, any other weapon should not be able to hit harder

**Comment by bonii-xx** [comment]
> the rapier can have paper parts for an extra level of smite.

You can also put paper onto the crossbow itself and melee things with it. Your point?

> Should, should, should

What if I told you, they should not? If you don't like how the crossbow feels, there are 3 other ranged weapons available, nobody is forcing you to use a crossbow.

**Comment by KnightMiner** [comment]
Closing due to lack of response and as I cannot see an actual issue in this report, just different expectations on how crossbows should behave.
